index.js

```diff
@@ -566,17 +566,18 @@ const createJavaBom = async (
         }
       }
     } else {
-      // Attempt to create dependency graph via global plugin. Very limited support.
-      let SBT_CMD = process.env.SBT_CMD || "sbt";
```

Comment:
Can we retain this since this variable might be used by someone?

Reply:
done